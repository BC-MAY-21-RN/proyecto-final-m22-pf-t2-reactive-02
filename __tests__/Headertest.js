import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../src/components/atoms/Header';

describe('Test from Header component', () => {
  it('Header snapshot test', () => {
    const renderHeader = renderer
      .create(<Header text={''} navigation={{}} />)
      .toJSON();
    expect(renderHeader).toMatchSnapshot();
  });
});
