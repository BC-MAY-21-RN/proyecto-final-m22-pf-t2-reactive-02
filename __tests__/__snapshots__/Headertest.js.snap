// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test from Header component Header snapshot test 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "#D3A6BA",
        "flexDirection": "row",
        "height": 60,
      }
    }
  >
    <RNGestureHandlerButton
      collapsable={false}
      exclusive={true}
      handlerTag={1}
      handlerType="NativeViewGestureHandler"
      onGestureEvent={[Function]}
      onGestureHandlerEvent={[Function]}
      onGestureHandlerStateChange={[Function]}
      onHandlerStateChange={[Function]}
      rippleColor={0}
      touchSoundDisabled={false}
    >
      <View
        accessible={true}
        collapsable={false}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "overflow": "hidden",
            }
          }
        >
          <View
            style={
              Object {
                "marginHorizontal": 15,
                "marginTop": 19,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                allowFontScaling={false}
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#242424",
                      "fontSize": 24,
                    },
                    Object {
                      "backgroundColor": "transparent",
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                î˜œ
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RNGestureHandlerButton>
    <Text
      style={
        Object {
          "color": "#263238",
          "fontSize": 22,
          "fontWeight": "bold",
          "marginVertical": 15,
        }
      }
    />
  </View>
</View>
`;
